### ISSUE TITLE
evil-jumper package fails to install

### ISSUE THREAD

**Issue by brenogil** [syl20bnr/spacemacs#5747]
#### Description
Package evil-jumper is failing me even on a fresh spacemacs install

Given that  `evil-jumper` is obsolete, should we simple remove it from spacemacs? 

Related to #3191?

#### Reproduction guide
Opening emacs. (even if for the first time after cloning syl20bnr/spacemacs repo)

*Observed behaviour:*

```
--> installing package: evil-jumper@spacemacs... [21/79]
Package evil-jumper is unavailable. Is the package name misspelled?
--> installing package: ws-butler@spacemacs-base... [79/79]
Found 1 new package(s) to install...
--> installing package: evil-jumper@spacemacs... [1/1]
Package evil-jumper is unavailable. Is the package name misspelled?
```

#### System Info
- OS: gnu/linux
- Emacs: 24.4.1
- Spacemacs: 0.105.17
- Spacemacs branch: master (rev. dbb6c4d)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm

**Comment by tzudot** [comment]
> evil-jumper has been integrated into evil-mode and is obsolete.

**Comment by syohex** [comment]
https://github.com/melpa/melpa/pull/3762

`evil-jumper` was removed from MELPA. Its features are merged into evil.
**develop** branch already removed `evil-jumper` but **master** branch does not apply that commit yet.

**Comment by StreakyCobra** [comment]
@syl20bnr Hotfix?

**Comment by floriangold** [comment]
Hi, there is a fix?

**Comment by adnissen** [comment]
Just ran into this as well on windows (obviously it's not platform dependent), but a fix would definitely be appreciated.

**Comment by nixmaniack** [comment]
Fix is in master now.

**Comment by StreakyCobra** [comment]
Fixed in master by bdf9fc584a5f6fdd4209fe979a110f61a440bafb and a9a68fb036e9c9c58bbb76978998530e37ec05db
